src/driver/find-files.h

```diff
@@ -8,6 +8,20 @@
 
 #include <api/compilation-data.h>
 
+struct yf_project_compilation_data {
```

Comment:
Could we declare this in `api` where all other top-level structs are declared?

Reply:
The reason I declared it here is that `yf_compilation_data` now performs the job of what `yf_project_compilation_data` used to do. I moved it to avoid accidentally using the wrong type, and this type is only used internally by the frontend now.

Reply:
Alright.